frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -30,13 +38,23 @@ const StyledLink = styled(Link)(({ theme }) => ({
 
 const GridContainer = styled(Box)(() => ({
     width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
As per the comment (but you'll also need to add `{theme}` to the style function)

```suggestion
    flexDirection: 'column',
    gap: theme.spacing(3)
```